hazenlib/HazenTask.py

```diff
@@ -10,21 +10,34 @@
 
 class HazenTask:
 
-    def __init__(self, data_paths: list, report: bool = False, report_dir: str = os.path.join(os.getcwd(), 'report')):
-        self.data_paths = sorted(data_paths)
+    def __init__(self, input_data, report: bool = False, report_dir = None):
+        # Check if input data is a single or list of files, load accordingly
+        if isinstance(input_data, list):
+            data_paths = sorted(input_data)
+            self.dcm_list = [dcmread(dicom)for dicom in data_paths]
+        else:
+            self.single_dcm = dcmread(input_data)
         self.report: bool = report
-        self.report_path = os.path.join(report_dir, type(self).__name__)
+        if report_dir is not None:
+            self.report_path = os.path.join(str(report_dir), type(self).__name__)
+        else:
+            self.report_path = os.path.join(os.getcwd(), 'report_image',
+                                            type(self).__name__)
```

Comment:
move report image output folder creation to here from init.py

Reply:
I think the default behaviour is consistently changed to have a `report_image` folder.
Motivation for renaming it is because these are not the actual reports that hazen output (the one that the physicists make after Excel manipulation), but contain images as the visual representation for the measurements. Folder name is consistent with the result dictionary key, making it easier to parse and locate files on the web-app backend.